timeline-chart/src/components/time-graph-annotation.ts

```diff
@@ -38,16 +39,24 @@ export class TimeGraphAnnotationComponent extends TimeGraphComponent {
     update(opts: TimeGraphAnnotationComponentOptions, style?: TimeGraphAnnotationStyle): void {
         if (opts) {
             this._options.position.x = opts.position.x;
-            this._options.position.y = this._row.position.y + (this._row.height * 0.5);
+            const align = this._style.verticalAlign;
+            this.updateYPosition(align);
         }
         if (style) {
```

Comment:
I can't find anywhere this method is called with a style parameter? Perhaps this can be removed?

Reply:
No I mean we always call update(opts) but never update(opts, style). So the if (style) is never used, I think.

Reply:
Fixed, I think.